collections/tuxlab.js

```diff
@@ -0,0 +1,40 @@
+var tuxlab = function(){}
```

Comment:
labfiles reference the tuxlab module. we can either publish it to npm or force them to use a relative path. I figured forcing a ./tuxlab.js path would be easiest..
or, if you'd prefer I can manipulate lab strings to reference the correct file. i.e., instructors won't add import and export I'll just add it to the beginning and end of the string. Your call